_posts/2019-10-17-how-to-achieve-cis-benchmark-compliance.adoc

```diff
@@ -737,48 +737,55 @@ Infrastructure as Code modules from Gruntwork.
 
-Gruntwork Compliance for CIS AWS Foundations Benchmark::
-  This guide also uses code from the link:https://gruntwork.io/achieve-compliance[Gruntwork CIS AWS
-  Foundations Benchmark repository], which contains the necessary configurations to achieve compliance.
-+
+==== Gruntwork Compliance for CIS AWS Foundations Benchmark
+This guide also uses code from the link:https://gruntwork.io/achieve-compliance[Gruntwork CIS AWS
+Foundations Benchmark repository], which contains the necessary configurations to achieve compliance.
 [.exceptional]
 IMPORTANT: You must be a [js-subscribe-cta]#Gruntwork Compliance subscriber# to access the Gruntwork
 Infrastructure as Code Library and the CIS AWS Foundations Benchmark modules.
 
 [[account_structure]]
-How to configure a production-grade AWS account structure::
-  Review the link:https://gruntwork.io/guides/foundations/how-to-configure-production-grade-aws-account-structure/[production-grade AWS account structure guide] to familiarize yourself with many of the concepts that this walkthrough depends on.
-
-
-Terraform::
-  This guide uses https://www.terraform.io/[Terraform] to define and manage all the infrastructure as code. If
-  you're not familiar with Terraform, check out
-  https://blog.gruntwork.io/a-comprehensive-guide-to-terraform-b3d32832baca[A Comprehensive Guide to Terraform],
-  https://training.gruntwork.io/p/terraform[A Crash Course on Terraform], and
-  link:/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library[How to Use the Gruntwork
-  Infrastructure as Code Library].
+==== How to configure a production-grade AWS account structure
+Review the link:https://gruntwork.io/guides/foundations/how-to-configure-production-grade-aws-account-structure/[production-grade AWS account structure guide] to familiarize yourself with many of the concepts that this walkthrough depends on.
+
+==== Terraform
+This guide uses https://www.terraform.io/[Terraform] to define and manage all the infrastructure as code. If
+you're not familiar with Terraform, check out
+https://blog.gruntwork.io/a-comprehensive-guide-to-terraform-b3d32832baca[A Comprehensive Guide to Terraform],
+https://training.gruntwork.io/p/terraform[A Crash Course on Terraform], and
+link:/guides/foundations/how-to-use-gruntwork-infrastructure-as-code-library[How to Use the Gruntwork
+Infrastructure as Code Library].
+
+==== Keybase (optional)
+As part of this guide, you will create IAM users, including, optionally, credentials for those IAM users. If you
+choose to create credentials, those credentials will be encrypted with a PGP key. You could provide the PGP keys
+manually, but a more manageable option may be to have your team members to sign up for https://keybase.io[Keybase],
+create PGP keys for themselves, and then you can provide their Keybase usernames, and the PGP keys will be retrieved
+automatically.
```

Comment:
Is it necessarily to include Terragrunt here, as well as any new repositories they need to create, or do you think it's sufficient that we link them to the LZ guide, which will mention those things?

I notice there's terragrunt commands mentioned here, so maybe at least mentioning terragrunt would be a good idea?

Reply:
Ah, I see that the `infrastructure-live` repo is "optional" in the diagram below.. will comment there...

Reply:
hi @rhoboat, we considered the following options:

1. We include a minor mention of Terragrunt in a paragraph.
2. We basically copy everything that Rho wrote and reword the necessary bits.
3. We do nothing and update the guide later to link to Rho's section once it's published.
4. We do nothing.

In the end, we decided to include a small paragraph (stolen from your work) on Terragrunt. In the future, we could simply de-duplicate work and link to anything you may publish in the LZ guide but I was hesitate to introduce a dependency for the moment.